README.md

```diff
@@ -2,17 +2,38 @@
 
 URDF Importer allows you to import a robot defined in [URDF](http://wiki.ros.org/urdf/XML) format in a Unity scene. URDF defines the geometry, visual meshes, kinematic and dynamic attributes of a Robot. Importer parses a URDF file and imports it into Unity using PhyX 4.0 articulation bodies.
 
-# Integrate URDF Importer into Unity Project
+# Using the Package
 
-- Clone or download the URDF Importer Repo
-- In your Unity project create a directory named `Plugins` in `Assets`
-- Copy the `URDFLibrary` and `UnityEditorScripts` directories into the newly created `Assets/Plugins` directory
-- Confirm the integration succeded by selecting `Assets` from the menu bar and look for the `Import Robot from URDF` option
+## Adding the URDF package
 
-# Importing a URDF into a Unity scene
+1. Create a new Project in your Unity Hub and open the Unity Editor.
```

Comment:
Do we need this point or can we prepend something like, 'In the Unity Editor,' to the second point?